spec.html

```diff
@@ -46482,7 +46482,7 @@ <h1>Valid Chosen Reads</h1>
           1. Let _readLen_ be the number of elements of _readValue_.
           1. If _chosenLen_ &ne; _readLen_, then
             1. Return *false*.
-          1. If _chosenValue_[_i_] &ne; _readValue_[_i_] for any integer value _i_ in the range 0 through _chosenLen_, exclusive, then
+          1. If _chosenValue_[_i_] &ne; _readValue_[_i_] for some integer _i_ in the range 0 &lte; _i_ &lt; _chosenLen_, then
```

Comment:
```suggestion
          1. If _chosenValue_[_i_] &ne; _readValue_[_i_] for some integer _i_ in the inclusive range from 0 to _chosenLen_, then
```

Reply:
this range is not inclusive on the right

Reply:
then can it use the parens form you’ve used elsewhere? (I’m sure in the rendering the difference would be easier to see than here0

Reply:
done